src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectModulesController.kt

```diff
@@ -0,0 +1,97 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.accelerator.db.ModuleStore
+import com.terraformation.backend.accelerator.model.EventModel
+import com.terraformation.backend.accelerator.model.ModuleModel
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.db.accelerator.EventId
+import com.terraformation.backend.db.accelerator.EventType
+import com.terraformation.backend.db.accelerator.ModuleId
+import com.terraformation.backend.db.default_schema.ProjectId
+import io.swagger.v3.oas.annotations.Operation
+import java.net.URI
+import java.time.Instant
+import java.time.LocalDate
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/modules")
+@RestController
+class ProjectModulesController(
+    private val moduleStore: ModuleStore,
+) {
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/projects/{projectId}")
+  @Operation(summary = "Gets modules for a project.")
+  fun listModules(
+      @PathVariable("projectId") projectId: ProjectId,
+  ): GetProjectModulesResponsePayload {
+    val models = moduleStore.fetchModulesForProject(projectId)
+    return GetProjectModulesResponsePayload(models.map { ProjectModule(it) })
+  }
+}
+
+data class ProjectModuleEventSession(
+    val id: EventId,
+    val startTime: Instant?,
+    val endTime: Instant?,
+    val meetingURL: URI?,
+    val recordingURL: URI?,
+    val slidesURL: URI?,
```

Comment:
We use `fooUrl` instead of `fooURL` elsewhere in the API, so probably good to rename these for consistency.